src/OpenTelemetry.Exporter.ZPages/ZPagesExporterStatsHttpServer.cs

```diff
@@ -121,9 +121,13 @@ private void WorkerThread()
                         using (var writer = new StreamWriter(output))
                         {
                             writer.WriteLine("Span Count : " + this.spanProcessor.GetSpanCount());
+                            writer.WriteLine("Error Count : " + this.spanProcessor.GetErrorCount());
+                            writer.WriteLine("Span Name\tStatus\t \tStart Timestamp\t \t \tEnd Timestamp\t \t \tLatency");
```

Comment:
please use `<tr><td>` syntax to build a table. So we will be able to apply CSS styles to this page.